### DISCUSSION TITLE
staticfile CDN 疑似被攻击者控制，是否考虑将文档代码示例里的资源引用都替换掉？

### DISCUSSION THREAD

**Discussion by mzlogin** [layui/layui#2070] [讨论]
参考 <https://v2ex.com/t/1053497>。

我看文档里大部分都是引用 unpkg 了，但还有少量推荐和引用 staticfile.net 的地方，比如：

- <https://layui.dev/docs/2/#install>

**Comment by sentsim** [comment]
只是部分资源吧。
 staticfile.net 有速度优势。

**Reply by Laiyiwen-02** [reply]
不至于吧，像国内的比如 www.zstatic.net 以及国际上的 unpkg 和 cdn.jsdelivr 还有 cdnjs 都挺不错啊，特别如果在国内的话，强推 www.zstatic.net，那才是真正的速度优势。
